lib/rexml/source.rb

```diff
@@ -65,9 +65,10 @@ class Source
     attr_reader :encoding
 
     module Private
+      SKIP_SPACES = /\s+/um
```

Comment:
About naming. How about `SPACES` `SPACES_PATTERN`?
I think most regexp constant names in REXML represents what it is or what it matches, but unlike other constants, the naming of `SKIP_SPACES` and `CUT_SPACES` focuses on action.

If it is only used in `def skip_spaces`, simply `@scanner.skip(/\s+/um)` might be good enough.

Reply:
Changed to `SPACES_PATTERN`.
Thank you!